File: code-wars-katas/staircase.py
# Function Description

# Complete the staircase function in the editor below.

# staircase has the following parameter(s):

# int n: an integer
# Print

# Print a staircase as described above.
import sys
def staircase(n):
    for row in range(n):
        a = n - 1 - row
        for col in range(n):
            if a <= col:
                sys.stdout.write('#')
            else:
                sys.stdout.write(' ')
        print()
          
staircase(4)

File: code-wars-katas/staircase.js
// Function Description

// Complete the staircase function in the editor below.

// staircase has the following parameter(s):

// int n: an integer
// Print

// Print a staircase as described above.

function staircase(n) {
  for (let i = 0; i < n; i++) {
    for (let j = 0; j < n; j++) {
      if (n - 1 - i <= j) {
        process.stdout.write("#");
      } else {
        process.stdout.write(" ");
      }
    }
    console.log();
  }
}

staircase(4);
